### PR TITLE
Remove all favorites us15

### PR DESCRIPTION
**Pull request by tony-gomes**
US 15

### CODE REVIEW

File: app/controllers/favorites_controller.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class FavoriteController < ApplicationController
+class FavoritesController < ApplicationController
```
Comment (mikez321) [inline app/controllers/favorites_controller.rb]: I'm glad these are working, we'll likely find a few more when we get this merged in with story 16

File: app/controllers/favorites_controller.rb [hunk 2]
```diff
@@ -18,7 +18,12 @@ def update
     pet = Pet.find(params[:pet_id])
     favorite.remove_pet(pet.id)
     session[:favorite] = @favorite.contents
-    flash[:pet_fav] = "Pet removed from favorites."
+    flash[:pet_fav_removed] = "Pet removed from favorites."
```
Comment (mikez321) [inline app/controllers/favorites_controller.rb]: Nice job on the updated flash messages!

File: app/views/favorites/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<h1>Your Favorite Pets</h1>
+<section>
+  <% if favorite.total_count > 0 %>
```
Comment (mikez321) [inline app/views/favorites/index.html.erb]: I can't think of where else this logic should live, but maybe something to follow up with during Monday's checkin

File: spec/features/favorites/destroy_all_favorites_spec.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  end
+
+  describe 'when I visit favorites index I see a link to remove all favorited pets' do
+    describe 'when I click that link I am redirected back to /favorites' do
+      it 'have no favorited pets is displayed & favorites indicator resets' do
+
+        visit '/'
+
+        within 'nav' do
+          # click_link "/favorites"
```
Comment (mikez321) [inline spec/features/favorites/destroy_all_favorites_spec.rb]: probably remove this unless it's there for a rason

File: spec/models/favorite_spec.rb [hunk 5]
```diff
@@ -10,6 +10,16 @@
     end
   end
 
+  describe "#check_pet" do
```
Comment (mikez321) [inline spec/models/favorite_spec.rb]: Seems like there were a few more model methods that didn't get tested here, let's just make sure they don't get forgotten.

Comment (tony-gomes) [reply spec/models/favorite_spec.rb]: could you list those out here explicitly so I can make a new card in projects?

File: app/views/favorites/index.html.erb (hunk 3 above)
Comment (tony-gomes) [reply app/views/favorites/index.html.erb]: ditto... I would like to revisit this during check-in to find out if there are better syntax options or other approaches for sure...

File: app/views/favorites/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    <% @favorited.each do |pet| %>
+      <div id="favorite-<%= pet.id %>" class="pet_block">
```
Comment (tony-gomes) [inline app/views/favorites/index.html.erb]: There was an accidental style tag that was here used in discussion... If your version has a style="display:flex" this is where it came from, and it has been removed...

File: app/controllers/favorites_controller.rb (hunk 1 above)
Comment (elguapogordo) [reply app/controllers/favorites_controller.rb]: yes, i did find a few more last night. they're fixed in the us16 branch

**Pull request merged**